brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-893 Improvement: Team - Social Links String Changes.

### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-893

### Screenshots

- https://prnt.sc/PyrGAqL3MEn6

### Types of changes




- Improvement

### How has this been tested?

- Drag and drop Team block.
- Visit the Social Links tabs and check the strings for social Icons.

### Checklist:
